content/docs/1.3.0/advanced-resources/deploy/customizing-default-settings.md

```diff
@@ -3,22 +3,24 @@ title: Customizing Default Settings
 weight: 1
 ---
 
-You may customize Longhorn's default settings when deploying it. You may specify, for example, `Create Default Disk With Node Labeled` and `Default Data Path` before starting Longhorn.
-
-This default setting is only for a Longhorn system that hasn't been deployed. It has no impact on an existing Longhorn system. The settings for any existing Longhorn system should be modified using the Longhorn UI _or_ by kubectl. Reference to the [knowledge base doc](https://longhorn.io/kb/troubleshooting-default-settings-do-not-persist/) for the details.
+You may customize Longhorn's default settings while installing or upgrading. You may specify, for example, `Create Default Disk With Node Labeled` and `Default Data Path` before starting Longhorn.
 
 The default settings can be customized in the following ways:
 
-- [Deploy](#deploy)
+- [Installation](#installation)
   - [Using the Rancher UI](#using-the-rancher-ui)
   - [Using the Longhorn Deployment YAML File](#using-the-longhorn-deployment-yaml-file)
   - [Using Helm](#using-helm)
-- [Upgrade](#upgrade)
+- [Update Settings](#update-settings)
   - [Using the Longhorn UI](#using-the-longhorn-ui)
   - [Using Kubectl](#using-kubectl)
   - [Using Script](#using-script)
+- [Upgrade](#upgrade)
```

Comment:
Can we also mention Rancher UI as well for upgrade and update? pair with what we have in installation.